bindings/dart/example/lib/data_provider/web.dart

```diff
@@ -24,12 +24,14 @@ Future<SetupData> getInputData() async {
 
   for (var asset in getAssets(features: features).entries) {
     final path = joinPaths([_baseAssetUrl, asset.value.urlSuffix]);
-    // We also load the wasm script here in order to check its integrity/checksum.
+    // We also load the wasm/worker script here in order to check its integrity/checksum.
     // The browser keeps it in cache so `injectWasmScript` does not download it again.
     final data = await _fetchAsset(path, asset.value.checksum.checksumSri);
 
     if (asset.key == AssetType.wasmScript) {
       await injectWasmScript(path);
+    } else if (asset.key == AssetType.webWorkerScript) {
+      fetched.putIfAbsent(asset.key, () => path);
```

Comment:
I would like to keep it as it is because in the following PR (#280) I can change it back to two branches. But yes, otherwise we could rewrite it as a `switch-case`.